_posts/2022-01-14-quorn-a-story-about-single-cell-protein.md

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Quorn™: a story about Single Cell Protein'
+image: assets/images/quorn/cover.png
+tags:
+- featured
+- submerged-fermentation
+- bioreactor
+- fungi
+- sticky
+categories:
+- industry
+author: keith
+---
+
+What if you could make food out of just basic chemicals and microbes - completely bypassing the conventional agricultural system, along with its inefficient land, water, chemical and energy use? 
```

Comment:
I actually think this is backwards. Crop farming is incredibly efficient compared to bioreactor SCP, and getting better. 

 - Land use: https://twitter.com/_HannahRitchie/status/1481576255086645248
 - Energy is exogenous to the system
 - Fertilizer is cheap-ish

It's not perfect, and crops are sensitive to external factors like weather and pests, but its a well studied problem. 

---- 
I'm bullish on bioreactor-derived milk casein ala Perfect Day, but I'm even _more_ bullish on soy-plant-derived casein ala [Nobell Foods](https://www.fastcompany.com/90657375/this-startup-just-raised-75-million-to-make-a-more-cheesy-plant-based-cheese). Soy farming is a "solved" problem, imo.

Reply:
Totally agree that crops > SCP. I wonder if there's another way to introduce the idea of SCP, or just pitch it from the protein perspective. Didn't know about Nobell Foods - I'll check it out! Let me think of a way to rephrase the intro.